bootstrap.sh

```diff
@@ -13,47 +13,43 @@ rm -f swig/vipsCC/*.cxx
 rm -f swig/vipsCC/VImage.h
 rm -f swig/vipsCC/VImage.py python/vipsCC/VError.py python/vipsCC/VMask.py python/vipsCC/Display.py
 rm -f benchmark/temp*
-( mkdir poop ; \
-	mv doc/libvips-docs.xml.in poop ; \
-	mv doc/Makefile.am poop ; \
-	mv doc/images poop ; \
-	mv doc/*.xml poop ; \
-	mv doc/*.py poop ; \
-	rm -rf doc/* ; \
-	mv poop/* doc ; \
-	rmdir poop \
-)
+find doc -depth \( \
+      -path doc/libvips-docs.xml.in \
+   -o -path doc/Makefile.am \
+   -o -path 'doc/images/*' \
+   -o -name '*.xml' ! -name libvips-docs.xml ! -path 'doc/xml/*' \
```

Comment:
The generated libvips-docs.xml was previously preserved, because it matched `*.xml`.